bspump/ipc/datagram.py

```diff
@@ -32,8 +32,9 @@ def __init__(self, app, pipeline, id=None, config=None):
 		# Receive Buffer Size
 		self.ReceiveBufferSize = int(self.Config['receiver_buffer_size'])
 
-		if ":" in self.Address:
-			host, port = self.Address.rsplit(":", maxsplit=1)
+		addrline = self.Address.strip()
+		if " " in addrline:
+			host, port = self.Address.rsplit(" ", maxsplit=1)
```

Comment:
This will be more tricky, since you have following variants:

- 1.2.3.4 1234
- 1.2.3.4:1234
- 1:2:3:4::1 1234

The last one is the IPv6 with port ... this one cannot use `:` because of the conflict with IPv6 format.